lib/handlerTemplate.js

```diff
@@ -192,7 +192,11 @@ function prepareRequestChain(conf) {
         if (requestConf.request) {
             template = new Template(requestConf.request);
             requestSpec.request = function(restbase, context) {
-                var req = template.eval(context);
+                var req = template.expand(context);
+                if (!requestConf.request.method) {
+                    // Do we really want this?
+                    req.method = context.request.method || 'get';
```

Comment:
Right, so we do agree that if the method is `GET` there should be no need for the user to specify the `method:` stanza in the config. Why then:

> 1) make all non-GET methods explicit in the specs (POST primarily);

?